node/src/components/consensus/metrics.rs

```diff
@@ -46,13 +55,15 @@ impl ConsensusMetrics {
     pub(crate) fn finalized_block(&mut self, finalized_block: &FinalizedBlock) {
         let time_since_proto_block = finalized_block.timestamp().elapsed().millis() as f64;
         self.finalization_time.set(time_since_proto_block);
+        self.time_of_last_finalized_block
+            .set(finalized_block.timestamp().millis() as i64);
         self.finalized_block_count.inc();
```

Comment:
While you're at it – can you change `finalized_block_count` to `IntGauge` and use the `.finalized_block.height()`? Otherwise, this metric does not survive restart.

Reply:
ping.

Reply:
Will do.

Reply:
Thanks